drivers/segger/config/SEGGER_RTT_Conf.h

```diff
@@ -77,6 +77,20 @@
 
 #define SEGGER_RTT_MEMCPY_USE_BYTELOOP  0
 
+/* RTT lock configuration */
+
+#ifndef __ASSEMBLY__
+extern spinlock_t g_segger_lock;
+#endif
+
+/* Lock RTT (nestable)   (i.e. disable interrupts) */
+
+#define SEGGER_RTT_LOCK()               irqstate_t __flags = spin_lock_irqsave(&g_segger_lock)
```

Comment:
Do you make changes without verifying them? Please run the relevant tests of Segger Sysview and upload the test screenshots

Reply:
If you can't provide test result, then revert the change in #15413